ownership-chain/runtime/src/precompiles/mod.rs

```diff
@@ -26,6 +27,27 @@ where
 	pub fn used_addresses() -> [H160; 7] {
 		[hash(1), hash(2), hash(3), hash(4), hash(5), hash(1025), hash(1027)]
 	}
+
+	fn is_delegatecall_to_custom_precompile(
+		&self,
+		code_address: H160,
+		context_address: H160,
+	) -> bool {
+		// Check if the code address is a precompile
+		if let IsPrecompileResult::Answer { is_precompile, .. } =
+			self.is_precompile(code_address, u64::MAX)
```

Comment:
why `u64::MAX` ? could it be removed as we don't use it within `is_precompile` fn?

Reply:
it's not possible to remove the `_gas` parameter of the function `is_precompile` cause it is declared in trait `PrecompileSet`